app/markets/MarketDetails.tsx

```diff
@@ -65,9 +73,44 @@ export const MarketDetails = ({ id }: MarketDetailsProps) => {
               <OutcomeBar market={market} />
             </div>
           </div>
-          {!marketModel.isClosed && (
+          <div className="px-4">
```

Comment:
I think is missing a bit of bottom padding so tabs are a bit more far away from the swapbox.

Reply:
I'll experiment